docs/MAINTAINERS-guidelines.md

```diff
@@ -0,0 +1,51 @@
+# Introduction
+All Hyperledger projects MUST have a `MAINTAINERS.md` file at the top-level directory of the source code. This document will provide guidelines on what should be included in the `MAINTAINERS.md` file.
+
+# Guidelines
+## List of Project Maintainers
+The first thing that MUST be included in the `MAINTAINERS.md` file is a list of the project's maintainers, both active and emeritus.
+
+It is recommended that the lists be sorted alphabetically and contain the maintainers name, GitHub ID, LFID, Company Affiliation, and Scope. The following shows the suggested format for the information:
+
+**Active Maintainers**
+
+| Maintainer | GitHub ID | LFID | Company Affiliation | Scope |
+| ---------- | --------- | ---- | ------------------- | ----- |
+|            |           |      |                     |       |
+
+**Emeritus Maintainers**
+
+| Maintainer | GitHub ID | LFID | Company Affiliation | Scope |
+| ---------- | --------- | ---- | ------------------- | ----- |
+|            |           |      |                     |       |
+
+## What Does Being a Maintainer Entail
+The `MAINTAINERS.md` file SHOULD contain information about the different types of maintainers that exist (whole project, repo, part of repo) and what their duties are (e.g., maintainers calls, quarterly reports, code reviews, issue cleansing).
+
+## How to Become a Maintainer
```

Comment:
I'm wondering whether we should allow for that info to be elsewhere and merely have a link to it in that case because several projects have that info in their docs today. On the other hand, the change this involves is simple and a link could be made the other way around...

Reply:
I was hoping the SHOULD gave that out.

Reply:
Yeah but the thing is that we do want to be able to get to that info from the MAINTAINERS file. This is often not the case today. So I think adding something stating that this needs to be in the file or linked from it would give better results.

Reply:
We should discuss with rest of @hyperledger/tsc. I was trying to not be too prescriptive with the guidelines.

Reply:
I'm sympathetic to Tracy's POV here